ext/ffi/lib.rs

```diff
@@ -766,6 +768,24 @@ fn is_fast_api_rv(rv: NativeType) -> bool {
   )
 }
 
+#[cfg(target_os = "linux")]
```

Comment:
Let's add a todo comment: `// TODO(@littledivy): Don't use normal heap with mprotect + PROT_EXEC for executable code to support SELinux.` to somehow make this optimization work in the future

Reply:
What sort of way would there be to make this work without normal heap + mprotect?

Reply:
mmap